### PR TITLE
Add solution to Two-Sum code challenge (via leetcode)

### PR DESCRIPTION
**Pull request by belloibrahv**
- Added all required files and scripts for the `Two Sum`  code challenge
- Wrote a discriptive algorithm for the solution.
- Use the Time complexity time based approach to solve problem

### CODE REVIEW

**Comment by abdulfataiaka** [comment]
Raise this pull request against the main branch of the `forked` repository in your account and not against the main branch of `abdulfataiaka/algorithms`

**Comment by abdulfataiaka** [comment]
Let's change the folder name of this algorithm from `sumTwo` to `twoSum` since that's the name of the algorithm

File: src/sumTwo/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+      const sum = first + second;
+      if ( sum === target ) {
+        return [i, j];
+      }
+    }
+  }
+}
+
+export default twoSum;
```
Comment (abdulfataiaka) [inline src/sumTwo/index.js:18]: Is there a solution with better time complexity than O(N^2)?

File: src/sumTwo/index.test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  test('test-3: To return two index that sums to 6', () => {
+    expect(twoSum([3,2,4], 6)).toEqual([1,2]);
+  })
+})
+
+describe('twoSum', () => {
+  test('test-4: To return two index that sums to 6', () => {
+    expect(twoSum([3,3], 6)).toEqual([0,1]);
+  })
+})
```
Comment (abdulfataiaka) [inline src/sumTwo/index.test.js:24]: You don't need to have multiple `describe blocks`. Just have a single `describe block` and multiple `test block` in it

Comment (abdulfataiaka) [reply src/sumTwo/index.test.js:24]: You don't need to say `test('test-n: To return...'`, you can just say `test('returns ...'`

File: src/sumTwo/index.js (hunk 1 above)
Comment (belloibrahv) [reply src/sumTwo/index.js:18]: Certainly, I would figure it out, as soon as I can

**Pull request closed without merging**